### ISSUE TITLE
its not installing on laravel 8, getting error TypeError

### ISSUE THREAD

**Issue by deepak-a-insnapsys** [vuongxuongminh/laravel-async#20]
I'm using laravel 8 framework. I'm getting the following error while installation using this command **_"composer require vxm/laravel-async"_**

> @php artisan ide-helper:generate

   TypeError 

  Argument 1 passed to Spatie\Async\Pool::concurrency() must be of the type int, null given, called in /var/www/html/vendor/vxm/laravel-async/src/AsyncServiceProvider.php on line 72

  at vendor/spatie/async/src/Pool.php:77
     73▕ 
     74▕         return $this;
     75▕     }
     76▕ 
  ➜  77▕     public function concurrency(int $concurrency): self
     78▕     {
     79▕         $this->concurrency = $concurrency;
     80▕ 
     81▕         return $this;

      +36 vendor frames 
  37  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
Script @php artisan ide-helper:generate handling the post-update-cmd event returned with error code 1

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

![image](https://user-images.githubusercontent.com/71549629/123513345-66a7f600-d6aa-11eb-9876-5d60a35993e5.png)

**Comment by vuongxuongminh** [comment]
What's Laravel version are you using? Please check in your config dir if have `async.php` file, delete it and try again.

**Comment by deepak-a-insnapsys** [comment]
**I'm using laravel 8** 
![image](https://user-images.githubusercontent.com/71549629/123581472-a03b4700-d7f9-11eb-97e7-fe6a35a9aa76.png)

I've checked the config directory but there is no async.php file.

_**Output of "composer require vxm/laravel-async"**_

Using version ^2.1 for vxm/laravel-async
./composer.json has been updated
Running composer update vxm/laravel-async
Loading composer repositories with package information
Updating dependencies
Lock file operations: 2 installs, 0 updates, 0 removals
  - Locking spatie/async (1.5.3)
  - Locking vxm/laravel-async (2.1.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Nothing to install, update or remove
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi
Discovered Package: arcanedev/log-viewer
Discovered Package: barryvdh/laravel-debugbar
Discovered Package: barryvdh/laravel-ide-helper
Discovered Package: beyondcode/laravel-websockets
Discovered Package: darkghosthunter/laraguard
Discovered Package: facade/ignition
Discovered Package: fideloper/proxy
Discovered Package: fruitcake/laravel-cors
Discovered Package: jamesmills/laravel-timezone
Discovered Package: lab404/laravel-impersonate
Discovered Package: langleyfoxall/laravel-nist-password-rules
Discovered Package: laravel/socialite
Discovered Package: laravel/tinker
Discovered Package: laravel/ui
Discovered Package: livewire/livewire
Discovered Package: nesbot/carbon
Discovered Package: nunomaduro/collision
Discovered Package: rachidlaasri/laravel-installer
Discovered Package: rappasoft/laravel-livewire-tables
Discovered Package: rappasoft/lockout
Discovered Package: spatie/laravel-activitylog
Discovered Package: spatie/laravel-permission
Discovered Package: stancl/tenancy
Discovered Package: tabuna/breadcrumbs
Discovered Package: torann/geoip
Discovered Package: vxm/laravel-async
Discovered Package: yajra/laravel-datatables-oracle
Discovered Package: yajra/laravel-oci8
Package manifest generated successfully.
112 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
> @php artisan ide-helper:generate

   TypeError 

  Argument 1 passed to Spatie\Async\Pool::concurrency() must be of the type int, null given, called in /var/www/html/vendor/vxm/laravel-async/src/AsyncServiceProvider.php on line 72

  at vendor/spatie/async/src/Pool.php:77
     73▕ 
     74▕         return $this;
     75▕     }
     76▕ 
  ➜  77▕     public function concurrency(int $concurrency): self
     78▕     {
     79▕         $this->concurrency = $concurrency;
     80▕ 
     81▕         return $this;

      +36 vendor frames 
  37  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
Script @php artisan ide-helper:generate handling the post-update-cmd event returned with error code 1

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
